WoW-Realm-Tracker/ViewControllers/RootViewController.swift

```diff
@@ -20,7 +21,7 @@ class RootViewController: UIViewController {
 }
 
 extension RootViewController: RealmsDelegate {
-    func receivedRealms(realms: [Realm]) {
-        transitionToFavorites(realms)
+    func receivedRealms(response: Result<[Realm], NSError>) {
+        response.map(transitionToFavorites)
```

Comment:
again, I'm not a _huge_ fan of using `map` for side effects, but don't let that stop you.